subprojects/docs/src/docs/userguide/troubleshootingDependencyResolution.adoc

```diff
@@ -20,21 +20,12 @@ Managing dependencies in a project can be challenging. This chapter describes te
 [[sub:version_conflicts]]
 === Resolving version conflicts
 
-Conflicting versions of the same module should be detected and either resolved or cause an exception. If you don't use transitive dependency management, version conflicts are undetected and may influence the order of the classpath determining what version of a dependency will be selected. In short, version conflicts can cause classpath issues and are extremely hard to debug.
-
-Gradle can <<managing_transitive_dependencies,manage transitive dependencies>> for you with the help of the following conflict resolution strategies:
-
-* _Newest_: The newest version of the dependency is used. This is Gradle's default strategy, and is often an appropriate choice as long as versions are backwards-compatible.
-* _Fail_: A version conflict results in a build failure. This strategy requires all version conflicts to be resolved explicitly in the build script. Fore more information, see api:org.gradle.api.artifacts.ResolutionStrategy#failOnVersionConflict()[].
-
-While the strategies introduced above are usually enough to solve most conflicts, Gradle provides more fine-grained mechanisms to resolve version conflicts:
+Gradle resolves version conflicts by picking the newest version of a module. link:https://scans.gradle.com/get-started[Build scans] and the <<sec:identifying_reason_dependency_selection,dependency insight report>> are immensely helpful in identifying why a specific version was selected. If the resolution result is not satisfying (e.g. the selected version of a module is too high) or it fails (because you configured api:org.gradle.api.artifacts.ResolutionStrategy#failOnVersionConflict()[]) you have the following possibilities to fix it.
```

Comment:
Nitpick: 'newest' sounds like we might take that latest release from the repo. 

'...the newest version of a module' -> '...the highest version of the conflicting ones.'

Reply:
done